Cargo.toml

```diff
@@ -70,6 +71,10 @@ tcp = [
     "tokio/tcp",
     "tokio/time",
 ]
+log = [
+    "tracing/log",
+    "tracing/std"
```

Comment:
i suspect that `tracing/std` can always be on, but i don't actually think _hyper_ needs any of the functionality behind that feature, so we really don't care if it's enabled or not

Reply:
I think it might be off is `default-features` is set to false; I suspect that pulling in syn + quote via instrument might be a bigger issue for hyper. In either case, I don't care _too_ much.